### ISSUE TITLE
Proposal: support new request to writer

### ISSUE THREAD

**Issue by gonejack** [cavaliercoder/grab#87]
It will be more easy to migrate from code like `io.Copy(writer, resp.Body)` to `grab` if `grab` supports new download to an opened writer.

```golang
grab.NewRequestToWriter(writer io.Writer, url string)
```

**Comment by cellargalaxy** [comment]
yes, it is a good feature, i would also like to support this feature

**Comment by opensaucerer** [comment]
@gonejack @cellargalaxy check this out. I just added support for `grab.NewRequestToWriter`.
https://github.com/opensaucerer/grab#this-extended-version-of-the-original-cavaliergophergrab-now-allows-you-to-download-files-into-a-given-iowriter
